From 3cfc6d254d067177d7b226854f4ccd1d0768ac98 Mon Sep 17 00:00:00 2001
From: Thiago Macieira <thiago.macieira@intel.com>
Date: Tue, 28 Jan 2020 14:32:06 -0800
Subject: [PATCH] qfloat16: include the tables in AVX2 binaries regardless

Having a QtCore build optimized for AVX2 does not imply all user
binaries and libraries are optimized the same way. Most of them will
actually have been built for the base platform, which means they require
access to these tables to operate if they are using qfloat16.

Change-Id: If79a52e476594446baccfffd15ee2da9e3693cce
---
 src/corelib/global/qfloat16tables.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/corelib/global/qfloat16tables.cpp b/src/corelib/global/qfloat16tables.cpp
index 3d764937d7..b87986d6b8 100644
--- a/src/corelib/global/qfloat16tables.cpp
+++ b/src/corelib/global/qfloat16tables.cpp
@@ -44,7 +44,7 @@
 
 QT_BEGIN_NAMESPACE
 
-#if !defined(__F16C__) && !defined(__ARM_FP16_FORMAT_IEEE)
+#if !defined(__ARM_FP16_FORMAT_IEEE)
 
 const quint32 qfloat16::mantissatable[2048] = {
 0,
-- 
2.25.0

